pkgs/by-name/ze/zeus-cli/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchzip,
+  buildNpmPackage,
+  makeWrapper,
+  nodejs,
+}:
+buildNpmPackage rec {
+  pname = "zeus-cli";
+  version = "1.6.6";
+
+  src = fetchzip {
+    url = "https://registry.npmjs.org/@zeppos/zeus-cli/-/zeus-cli-${version}.tgz";
+    hash = "sha256-+S3vtn6R3sWw+AOMZsYpTjzBtf3se7WIlNIngDbr4b4=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ nodejs ];
+
+  npmDepsHash = "sha256-s1blLke85tjFMJQNLP/Gsb9pREGWuFBKbjJuRDQ1srY=";
+
+  patches = [
+    ./fix_script_name.patch
+    ./fix_writable.patch
+  ];
+
+  postPatch = ''
+    cp ${./package-lock.json} package-lock.json
+  '';
+
+  dontNpmBuild = true;
+
+  postInstall = ''
+    # Add npm to PATH (needed to create and compile apps)
+    wrapProgram "$out/bin/${meta.mainProgram}" \
```

Comment:
This shouldn't be mainProgram the same as it shouldn't be pname. Please write the literal string here